### ISSUE TITLE
Your Page missing two images

### ISSUE THREAD

**Issue by Brennername** [sqlitebrowser/dbhub.io#384]
Two images on the Your Page (dbhub.io/UserName) are missing or can't load for some reason.

**Comment by justinclift** [comment]
Oh, that's unexpected.  In trying that exact url just now (using Firefox), I'm not seeing missing images so I'm not sure which ones you're meaning.

Would you be ok to screenshot it?

**Comment by Brennername** [comment]
Sorry it must have been temporary, and I have no idea how I landed on the page.

**Comment by justinclift** [comment]
No worries at all @Brennername. :smile:

**Comment by Brennername** [comment]
### Ok. Update on this issue, it's reproduceable.

- I found out the missing image was https://lh3.googleusercontent.com/   (the rest redacted)

- This means it's not fetching the image from my profile on google. When I used github to auth, it did not have a problem. I did not mention and had forgotten that I used gmail the first time.

- I discovered this because I had an error that my gmail email had already been associated with another account after logging in via github on the auth. So I logged out and selected google for the auth provider and it took me to the "Home" route and that's where the images do not appear.

- I am only assuming it's my profile picture because that is naturally what should appear where those images try to load. 

**Screen shot available:**
![Screenshot_20250106_095816](https://github.com/user-attachments/assets/a2cd65be-6b69-4bfb-9a52-ed0aeb0dd44a)

**Comment by justinclift** [comment]
As a data point, we'll likely be sunsetting the dbhub.io platform some time in the coming weeks.  It's mostly just waiting on myself to have sufficient time to think through all the right steps, write up the sunset announcement info + do blog posts, etc.  And figure out what a good notice period is for people (60 days maybe?).

**Comment by Brennername** [comment]
Are you replacing it with anything?

**Comment by FriedrichFroebel** [comment]
This will probably be part of the announcement mentioned above, but as far as I am aware, there are no plans from our side to provide a similar platform as replacement.

**Comment by justinclift** [comment]
> Are you replacing it with anything?

Nope.  It was a failed attempt to make a service people would find useful and pay for, but unfortunately never got enough usage to even be worth hooking up a payment system. 🤷‍♂
